### PR TITLE
update docs related to partition table dynamic mode

### PR DESCRIPTION
**Pull request by qw4990**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -582,11 +582,11 @@ Query OK, 0 rows affected (0.09 sec)
 
 > **警告：**
 >
-> 目前分区表动态模式为实验特性，不建议在生产环境中使用。
+> 目前分区表动态裁剪模式为实验特性，不建议在生产环境中使用。
 
 - 作用域：SESSION | GLOBAL
 - 默认值：static
-- 这个变量用来设置是否开启分区表动态模式。关于动态模式的详细说明请参阅[分区表动态模式](/partitioned-table.md#动态模式)。
+- 这个变量用来设置是否开启分区表动态裁剪模式。关于动态裁剪模式的详细说明请参阅[分区表动态裁剪模式](/partitioned-table.md#动态裁剪模式)。
```
Comment (qiancai) [inline system-variables.md:589]:
```suggestion
- 这个变量用来设置是否开启分区表动态裁剪模式。关于动态裁剪模式的详细说明，请参阅[分区表动态裁剪模式](/partitioned-table.md#动态裁剪模式)。
```

**Review by qiancai — APPROVED** [review]

**Comment by TomShawn** [comment]
@qw4990 Please resolve the conflicts.
@lysu @rebelice @tiancaiamao PTAL

**Review by lysu — APPROVED** [review]

**Comment by qw4990** [comment]
All conflicts are resolved, PTAL @TomShawn @tiancaiamao

**Pull request merged**